.husky/pre-commit

```diff
@@ -7,5 +7,5 @@
 # Now you can use npm or any Node.js commands
 
 # npm run lint
-npm run lint -- -- --fix
-npm run format
+#npm run lint -- -- --fix
```

Comment:
I just leave a smile here :) if you need to ignore husky next time, please do it with env variable instead

Reply:
👍